docs/dictionary/property/patterns.lcdoc

```diff
@@ -21,29 +21,41 @@ Example:
 set the patterns of me to storedPatternsList
 
 Value:
-The <patterns> of an <object> is a list of pattern specifiers, one per <line>.  A pattern specifier is a built-in pattern number between 1 and 164 (corresponding to LiveCode's built-in patterns 136 to 300), or the <ID> of an <image> to use for a pattern. LiveCode looks for the specified <image> first in the <current stack>, then in other open <stacks>.
-The <patterns> of an <object> contains eight <lines>, some of which may be empty.
+The <patterns> of an <object> is a list of pattern specifiers, one per <line>. A pattern specifier 
+is a built-in pattern number between 1 and 164 (corresponding to LiveCode's built-in patterns 136 to 
+300), or the <ID> of an <image> to use for a pattern. LiveCode looks for the specified <image> first 
+in the <current stack>, then in other open <stacks>.
+The <patterns> of an <object> consists of eight <lines>, some of which may be empty.
 
 Description:
-Use the <patterns> <property> to get all eight basic pattern <properties> at once, or to set the patterns of one <object> to be the same as the patterns of another <object>.
-
-You can set all these patterns individually; the <patterns> <property> simply provides a shorter method of dealing with all of them at once. Each <line> of the <patterns> corresponds to one of the following pattern <properties> :
-
-        * Line 1: the foregroundPattern
-        * Line 2: the <backgroundPattern> 
-        * Line 3: the hilitePattern
-        * Line 4: the <borderPattern> 
-        * Line 5: the topPattern
-        * Line 6: the <bottomPattern> 
-        * Line 7: the shadowPattern
-        * Line 8: the focusPattern
-
-If you leave a line blank when setting the <patterns>, the pattern <property> corresponding to that <line> is left unchanged.
-
-If the <patterns> <property> of an <object> reports a blank <line>, the corresponding pattern is not set for the individual <object>, but is <inheritance|inherited> from the <object|object's> <owner>. Use the form the effective patterns of <object> to obtain the patterns used for the object, whether set for the <object> or <inheritance|inherited>.
+Use the <patterns> <property> to get all eight basic pattern <properties> at once, or to set the 
+patterns of one <object> to be the same as the patterns of another <object>.
+
+You can set all these patterns individually; the <patterns> <property> simply provides a shorter 
+method of dealing with all of them at once. Each <line> of the <patterns> corresponds to one of the 
+following pattern <properties> :
+
+ * Line 1: the foregroundPattern
+ * Line 2: the <backgroundPattern> 
+ * Line 3: the hilitePattern
+ * Line 4: the <borderPattern> 
+ * Line 5: the topPattern
+ * Line 6: the <bottomPattern> 
+ * Line 7: the shadowPattern
+ * Line 8: the focusPattern
```

Comment:
@asayd Seeing as you're here, would you mind turning all of these patterns into links please?